scenes/world_map/frays_end.tscn

```diff
@@ -1,7 +1,7 @@
-[gd_scene load_steps=44 format=4 uid="uid://cufkthb25mpxy"]
+[gd_scene load_steps=45 format=4 uid="uid://cufkthb25mpxy"]
 
 [ext_resource type="TileSet" uid="uid://07fq3rspk8ia" path="res://scenes/tileset.tres" id="1_4evgk"]
-[ext_resource type="Script" uid="uid://8div00rgvsds" path="res://scenes/world_map/components/frays_end.gd" id="1_4gse6"]
+[ext_resource type="Script" path="res://scenes/world_map/components/frays_end.gd" id="1_4gse6"]
```

Comment:
I don't think we want to remove the UID here?

Reply:
No idea why that changed. Godot shenanigans. I'll revert that and merge. Thanks!!